internal/cluster/validations/validations.go

```diff
@@ -220,6 +221,20 @@ func ValidateBaseDNS(dnsDomainName, dnsDomainID, dnsProviderType string) error {
 	return validateBaseDNS(dnsDomainName, dnsDomainID, dnsProvider)
 }
 
+func ValidateNTPSource(ntpSource string) bool {
+	if addr := net.ParseIP(ntpSource); addr != nil {
+		return true
+	}
+
+	if hostName, err := net.LookupHost(ntpSource); err != nil {
```

Comment:
What if the NTP server is on-prem, not accessible to cloud.redhat.com?  The only thing we really want to test is if it's accessible to the hosts.

Reply:
Added for @filanov suggestion for validation https://github.com/openshift/assisted-service/pull/675#discussion_r525132199

Maybe it should be a regex validation.